content/docs/examples/multicluster/split-horizon-eds/index.md

```diff
@@ -0,0 +1,377 @@
+---
+title: Multicluster with Split Horizon EDS
+description: Example of leveraging Istio's Split Horizon EDS to create a Multi-Cluster.
+weight: 85
+keywords: [kubernetes,multicluster]
+---
+
+This example demonstrates how to use a variation of the [Istio multicluster](/docs/setup/kubernetes/multicluster-install/vpn/) together with _Split Horizon EDS_ (introduced in Istio 1.1) to join two Kubernetes clusters together to allow access to remote instances of a service via the remote Istio ingress gateway. By following the steps in this example you will setup a topology of two clusters as described in the following diagram:
+
+  {{< image width="80%" ratio="36.01%"
+  link="./diagram.svg"
+  caption="Istio mesh spanning multiple Kubernetes clusters with the Split Horizon EDS configured"
+  >}}
+
+The `local` cluster hosts the Istio Pilot and other Istio components while the `remote` cluster only runs the Istio Citadel, Sidecar Injector and Ingress gateway.
+However, unlike the requirements of [Istio multicluster](/docs/setup/kubernetes/multicluster-install/vpn/), no VPN connectivity is required nor direct network access between workload from different clusters.
+
```

Comment:
It would be useful to use this reasoning highlight why you use split EDS over the vpn.